.github/workflows/build-and-test.yml

```diff
@@ -54,6 +54,7 @@ jobs:
     - name: Run integration tests
       env: ${{matrix.env}}
       run: |
+        sudo sysctl net/netfilter/nf_conntrack_max=131072 # https://github.com/kubernetes-sigs/kind/issues/2240
```

Comment:
Saw a spurious error in github actions while booting KiND because of this, so just adding it to make GHA happy

Reply:
Thanks for clarifying, it helped :)